prow/apis/prowjobs/v1/types.go

```diff
@@ -163,14 +163,26 @@ type ProwJobSpec struct {
 	ReporterConfig *ReporterConfig `json:"reporter_config,omitempty"`
 
 	// RerunPermissions holds information about which users can rerun the job
-	RerunPermissions *RerunPermissions `json:"rerun_permissions,omitempty"`
+	RerunPermissions RerunPermissions `json:"rerun_permissions,omitempty"`
+}
+
+type GitHubTeamSlug struct {
+	Slug string `json:"slug"`
+	Org  string `json:"org"`
 }
 
 type RerunPermissions struct {
```

Comment:
Please get rid of this redundant struct: https://github.com/kubernetes/test-infra/blob/c9c1e11a31f82400250a47f50614e6b349f2a6ed/prow/config/config.go#L381-L389